src/Microsoft.Framework.Caching.Distributed/CacheExtensions.cs

```diff
@@ -3,6 +3,7 @@
 
 using System;
 using System.IO;
+using Microsoft.Framework.Internal;
```

Comment:
Yes the [NotNullAttribute] in the common package lives in this namespace. That's why I added this namespace after removing the `NotNullAttribute.cs` file in each of the projects.